plugins/woocommerce/client/admin/client/settings-payments/components/modals/modals.scss

```diff
@@ -92,7 +92,7 @@
 	}
 
 	&__actions {
-		flex-direction: row-reverse;
+		flex-direction: row-reverse; // use this to force items to start from end. That would require actions to be introduced in reverse order.
```

Comment:
Start with uppercase.

Reply:
thanks! addressed in https://github.com/woocommerce/woocommerce/pull/54263/commits/1cb4f8eb9290355d38f8cefebb9784d70bdf6450